### ISSUE TITLE
No error log showing when sending push to APNs 

### ISSUE THREAD

**Issue by Mathijs003** [Smile-SA/node-pushserver#63]
I'm testing my push notifications on an android and iOS device. 

I can send from the iOS device to my android device but not vice versa.

The iOS device has a token, and the certificates are set up.

When I send a push to my iOS device no logs are shown in the terminal of the node push server.

How can I get the response code and messages from the APNs service? 

Thanks in advance

**Comment by marob** [comment]
> I can send from the iOS device to my android device but not vice versa.
Notifications are **not** sent from a device, but from the pushserver.

I'm not sure to understand what you're doing.

**Comment by Mathijs003** [comment]
I'm sorry, my explanation is not correct.

In short: my iOS app isn't receiving notifications but my android app does.
In the console there are no logs printed after "listening on port 8000"
even when there must be a error response from apns because my app can't
receive notifications.

Tl;dr

Want to check the responses from the apns service

**Comment by marob** [comment]
We can't diagnose the problem without more accurate data.

I would say the problem can come from:
* certificates
* closed ports (https://support.apple.com/en-sa/HT203609)
* other?

Regards

**Comment by Mathijs003** [comment]
I'm sorry for wasting your time, please close or preferably delete this issue. 

The problem was I forgot to implement apns in the frontend.
